src/appleseed/renderer/modeling/aov/cryptomatte.cpp

```diff
@@ -402,7 +527,7 @@ namespace
             const AOVComponents&        aov_components,
             ShadingResult&              shading_result) override
         {
-            uint32_t m3hash = 0;
+            uint32 m3hash = 0;
```

Comment:
I can't comment further down in this method, but can you mark as `const` all variables that can be const?

Reply:
You can also precompute `1.0 / m_rcp_canvas_width` etc.